.cproject

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I'll just note that Eclipse project files are also artifacts that don't really belong in revision control :)